week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -6,18 +6,27 @@
 
   NOTE: only the names should be returned, not the means of transport.
 */
+// function journeyPlanner(select, vehicle) {
+//   let common = [];
+//   for (let i = 0; i < select.length; i++) {
+//     if (select[i].includes(vehicle)) {
+//       common.push(select[i][0]);
+//     }
+//   }
+//   return common;
+// }
 
-function journeyPlanner() {
-
+function journeyPlanner(arr, mean) { // mean is the vehicles
```

Comment:
Could we name this param vehicles instead of using a comment?

Reply:
That would be more meaningful.
Thanks